src/roadmap/releases.md

```diff
@@ -193,15 +205,23 @@ Change the `generate_params.toml` file in **each** new chart as follows:
   1. If there's a `[workflowManager]` section, change `versionsApiURL` to
      `"https://versions.deis.com"` and `doctorApiURL` to `"https://doctor.deis.com"`
 
-Change the `workflow-$WORKFLOW_RELEASE/tpl/deis-controller-deployment.yaml` file:
+Change the `workflow-$WORKFLOW_RELEASE/tpl/deis-controller-deployment.yaml` and
+`workflow-$WORKFLOW_RELEASE/tpl/deis-controller-rc.yaml` files:
```

Comment:
Well the instruction to remove `KUBERNETES_POD_TERMINATION_GRACE_PERIOD_SECONDS` was originally targeted at the RC manifest, so I grandfathered it in, but I guess it's irrelevant now. I removed it.